<launch>

<rosparam command="load" file="$(find coverage_planar_planner)/param/cage.yaml" />


<node
name='planner_node'
pkg='coverage_planar_planner'
type='planner_node.py'
/>

<node
name='simulator_node'
pkg='coverage_planar_planner'
type='simulator_node.py'
/>

<node
name='plotter_node'
pkg='coverage_planar_planner'
type='plotter_node.py'
/>

<node
name='rqt_service_caller'
pkg='rqt_service_caller'
type='rqt_service_caller'
/>

<!--node
name='plot_pose'
pkg='rqt_plot'
type='rqt_plot'
args='/pose/position /pose/yaw'
/-->

<!--node
name='plot_coverage'
pkg='rqt_plot'
type='rqt_plot'
args='/coverage'
/-->

</launch>
